src/libraries/System.Runtime/tests/System/Diagnostics/CodeAnalysis/ExperimentalAttributeTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Diagnostics.CodeAnalysis.Tests
+{
+    public class ExperimentalAttributeTests
+    {
+        [Fact]
+        public void TestConstructor()
+        {
+            var attr = new ExperimentalAttribute("diagnosticId");
+
+            Assert.Equal("diagnosticId", attr.DiagnosticId);
+            Assert.Null(attr.UrlFormat);
+        }
+
+        [Fact]
+        public void TestConstructor_throws_if_null()
+        {
+            Assert.Throws<ArgumentNullException>(() => new ExperimentalAttribute(diagnosticId: null));
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(" ")]
+        [InlineData("\t")]
+        public void TestConstructor_throws_if_empty_or_whitespace(string diagnosticId)
```

Comment:
Remove.

Whatever tooling is validating these attributes should also issue diagnostics for incorrect usage.